src/igvfd/schemas/changelogs/whole_organism.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog for *`whole_organism.json`*
 
+### Schema version 11
+
+* Remove `taxa` from `required`.
```

Comment:
I think it would be nice to indicate the property is not only not required but also calculated